Maui/MLTrainer/global.json

```diff
@@ -1,6 +1,5 @@
 {
   "sdk": {
-    "version": "6.0.300",
```

Comment:
unsure which specific net8 version we would like to point to/if we really need a specific version in this case?